web/libs/datamanager/src/components/Common/Userpic/Userpic.scss

```
@@ -0,0 +1,67 @@
+.userpic {
+  width: 28px;
+  height: 28px;
+  display: flex;
+  font-size: 12px;
+  position: relative;
+  align-items: center;
+  border-radius: 50%;
+  justify-content: center;
+  background: rgba(0, 0, 0, 0.08);
+  box-shadow: 0px 0px 0px 1px rgba(0, 0, 0, 0.1);
+
+  img {
+    opacity: 0;
+    width: 100%;
+    height: 100%;
+    font-size: 12px;
+    line-height: 22px;
+    object-fit: cover;
+    position: absolute;
+    border-radius: 100%;
+  }
+
+  &__username {
+    display: block;
+    line-height: 22px;
+    font-weight: bold;
+    opacity: 0.6;
+  }
+
+  &__badge {
+    position: absolute;
+
+    &_topLeft {
+      top: 0;
+      left: 0;
+    }
+
+    &_topRight {
+      top: 0;
+      right: 0;
+    }
+
+    &_bottomLeft {
+      bottom: 0;
+      left: 0;
+    }
+
+    &_bottomRight {
+      bottom: 0;
+      right: 0;
+    }
+  }
+
+  &_faded {
+    background: rgba(0, 0, 0, 0.03);
+    box-shadow: inset 0px 0px 0px 1px rgba(0, 0, 0, 0.1);
```

Flagging as opportunity for css var replacement. Further usage review needed.
```suggestion
    background: rgba(var(--black), 0.03);
    box-shadow: inset 0px 0px 0px 1px var(--black_10);
```